src/lib/Grammar.mly

```diff
@@ -66,30 +86,41 @@ sign:
   | d = decl; s = sign
     { d :: s }
 
-plain_atomic_cof_except_atomic_term:
-  | BOUNDARY t = term
+plain_atomic_in_cof_except_term:
+  | BOUNDARY t = atomic_term
```

Comment:
I decided to weaken the expressiveness in this particular case... unless people make another wish at #69.